### PR TITLE
Update 418.json

### PR DESCRIPTION
**Pull request by evilpacket**
npm received an update from the author that this was fixed in 0.1.6.

### CODE REVIEW

File: vuln/npm/418.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
...
   "module_name": "rgb2hex",
   "cves": [],
-  "vulnerable_versions": "<=0.1.0",
-  "patched_versions": null,
-  "recommendation": "No fix is currently available for this vulnerability.\n\nIt is our recommendation to not install or use this module at this time.",
+  "vulnerable_versions": "<0.1.6",
+  "patched_versions": >=0.1.6,
+  "recommendation": "Update to version 0.1.6 or later",
   "references": [
     "https://hackerone.com/reports/319629",
```
Comment (ChALkeR) [inline vuln/npm/418.json]: Add a ref to
`"https://github.com/christian-bromann/rgb2hex/commit/9e0c38594432edfa64136fdf7bb651835e17c34f",`
?

Comment (lirantal) [reply vuln/npm/418.json]: 👍

File: vuln/npm/418.json [hunk 2]
```diff
@@ -8,9 +8,9 @@
   "author": "Сковорода Никита Андреевич (https://github.com/ChALkeR)",
   "module_name": "rgb2hex",
   "cves": [],
-  "vulnerable_versions": "<=0.1.0",
-  "patched_versions": null,
-  "recommendation": "No fix is currently available for this vulnerability.\n\nIt is our recommendation to not install or use this module at this time.",
+  "vulnerable_versions": "<0.1.6",
+  "patched_versions": >=0.1.6,
```
Comment (ChALkeR) [inline vuln/npm/418.json]: Quotes are missing, this breaks JSON.

**Comment by evilpacket** [comment]
argh I'm bad at git apparently. `evilpacket-patch-1` was for some previous spelling changes for some other advisory. @pxlpnk

**Comment by lirantal** [comment]
No worries @evilpacket, I opened #349 so we can quickly resolve the issue

**Pull request closed without merging**